flink-tests/pom.xml

```diff
@@ -44,7 +44,14 @@ under the License.
 	</properties>
 
 	<dependencies>
-	
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-core</artifactId>
+			<version>2.17.1</version>
+			<scope>test</scope>
+			<type>test-jar</type>
```

Comment:
What do we need from the test jar?

Reply:
I guess that's not needed anymore after switching to Junit extension, I'll remove this dependency.